scripts/install_prereqs.sh

```diff
@@ -57,12 +59,63 @@ function install_secrets() {
   fi
 }
 
-if [[ ! -z $QUICKSTART ]]; then
+# apt_install program package
+#
+# Installs the given package if the given command is missing
+function apt_install() {
+  local program="$1"
+  local package="$2"
+  which "$program" >& /dev/null || sudo apt-get install "$package"
+}
+
+secrets_installed=false
+
+# Default values, if not supplied on the command line or environment
+platform="iOS"
+method="xcodebuild"
+
+if [[ $# -eq 0 ]]; then
+  # Take arguments from the environment
+  project=$PROJECT
+  platform=${PLATFORM:-${platform}}
+  method=${METHOD:-${method}}
+
+else
+  project="$1"
+
+  if [[ $# -gt 1 ]]; then
+    platform="$2"
+  fi
+
+  if [[ $# -gt 2 ]]; then
+    method="$3"
+  fi
+fi
+
+echo "Installing prerequisites for $project for $platform using $method"
+
+if [[ ! -z "${QUICKSTART:-}" ]]; then
   install_secrets
-  ./scripts/setup_quickstart.sh "$QUICKSTART"
+  scripts/setup_quickstart.sh "$QUICKSTART"
```

Comment:
Probably the bundler setup needs to be before this:

It's failing with 

```
/Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/spec_set.rb:91:in `block in materialize': Could not find CFPropertyList-3.0.1 in any of the sources (Bundler::GemNotFound)
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/spec_set.rb:85:in `map!'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/spec_set.rb:85:in `materialize'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/definition.rb:170:in `specs'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/definition.rb:237:in `specs_for'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/definition.rb:226:in `requested_specs'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/runtime.rb:108:in `block in definition_method'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/runtime.rb:20:in `setup'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler.rb:107:in `setup'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/bundler/setup.rb:20:in `<top (required)>'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/travis/.rvm/rubies/ruby-2.6.4/lib/ruby/site_ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
```

and later the quickstart run fails because the workspace wasn't created.

Reply:
Interesting! Thanks for finding that and apologies for not seeing that myself.

Reply:
That did the trick. This is now ready for another round of review.